src/components/Theme/constants.css

```diff
@@ -15,6 +11,11 @@
  */
 @unit: 1/16em;
 
+@small-viewport-breakpoint: 479*@unit;
+@smaller-viewport-breakpoint: 359*@unit;
+@custom-media --small-viewport (max-width: @small-viewport-breakpoint);
+@custom-media --smaller-viewport (max-width: @smaller-viewport-breakpoint);
```

Comment:
So, point of discussion:

Media queries are _root level_. What this means is that _the `em` that we refer to in our media queries can be different to the `em` that we refer to inside our container_!

E.g. if the client sets our container font size to `16px`, but sets the _root_ font size to `8px`, then we effectively end up in a world where we're doing

```css
@media(max-width: 100px) {
  width: 200px;
}
```

rather than

```css
@media(max-width: 200px) {
  width: 200px;
}
```

I don't think that any clients do this (most seem to set the root font-size, and setting our font-size separately would be a weird thing to do), and I've checked that the SDK doesn't break horribly if this sort of thing happens (our breakpoints just hit at different times to what we'd like)

Let me know if this doesn't make sense, might be easier to chat offline

But it's a very awkward point, that I can't see any great ways around 😢